<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortBrowser</class>
 <widget class="QWidget" name="SerialPortBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>243</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>239</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="statusEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="portsLabel">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="portsBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="baudRateLabel">
      <property name="text">
       <string>BaudRate:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QComboBox" name="baudRateBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="parityLabel">
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QComboBox" name="parityBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dataBitsLabel">
      <property name="text">
       <string>DataBits:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QComboBox" name="dataBitsBox"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="flowControlLabel">
      <property name="text">
       <string>FlowControl:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QComboBox" name="flowControlBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="stopBitsLabel">
      <property name="text">
       <string>StopBts:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QComboBox" name="stopBitsBox"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="refreshButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="disconnectButton">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
